addons/cert-manager/main.tf

```diff
@@ -0,0 +1,17 @@
+module "helm_addon" {
+  source = "../helm"
+
+  manage_via_gitops = var.manage_via_gitops
+  helm_config       = local.helm_config
+  addon_context     = var.addon_context
+
+  depends_on = [kubernetes_namespace_v1.this]
+}
+
+resource "kubernetes_namespace_v1" "this" {
+  count = try(local.helm_config["create_namespace"], true) && local.helm_config["namespace"] != "cert-manager" ? 1 : 0
+
+  metadata {
+    name = local.helm_config["namespace"]
+  }
+}
```

Comment:
we can remove this `kubernetes_namespace_v1` as it is being manged by `create_namespace` variable. [here](https://github.com/clouddrove/terraform-aws-eks-addons/blob/21f2277e1363033ab88742f9018ca0ab64710821/addons/cert-manager/locals.tf#L10)